### PR TITLE
Restore previous timeout after server started

### PR DESCRIPTION
**Pull request by meafmira**

### CODE REVIEW

File: upcloud/client/client.go [hunk 1]
```diff
@@ -3,12 +3,12 @@ package client
 import (
 	"bytes"
 	"fmt"
-	"github.com/blang/semver"
-	"github.com/hashicorp/go-cleanhttp"
```
Comment (Jalle19) [inline upcloud/client/client.go]: Is this really not used anymore?

Comment (meafmira) [reply upcloud/client/client.go]: @Jalle19 fixed

**Comment by Jalle19** [comment]
@iler you should configure credentials in Travis CI so the tests can be run

**Comment by iler** [comment]
@Jalle19 as this is a PR outside of the organization it does not allow the use of secret environment variables in the tests as explained here https://docs.travis-ci.com/user/pull-requests/#Pull-Requests-and-Security-Restrictions. So those are configured there but can not be used in this PR.

**Comment by Jalle19** [comment]
@iler ah, didn't know about that restriction!

**Pull request merged**